airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/task/internal/LoadPipelineStepTask.kt

```diff
@@ -221,6 +226,7 @@ class LoadPipelineStepTaskFactory(
         flushStrategy: PipelineFlushStrategy?,
         part: Int,
         numWorkers: Int,
+        taskIndex: Int,
```

Comment:
nonblocking: maybe nicer to use taskName:String? then we don't have magic numbers like -1, 2, etc. floating around, and is maybe also useful for future debugging